_documentation/05-User-Manual.md

```diff
@@ -131,9 +131,60 @@ ___
 
     ![Webp net-resizeimage (10)](https://user-images.githubusercontent.com/43112419/97261802-9c5bbd00-1845-11eb-9618-5907372c0755.png)
 
-    
+    Many manufacturers have options that cannot be translated into IPP attributes. So web interface provides the option to set up these options in this segment.
+
+    The printing defaults are automatically fetched for each job and user is not required to pass them each and every time using the command-line.
```

Comment:
The printing defaults are automatically fetched for each job and **the** user is not required to pass them each and every time using the command-line.